course/templates/course/course_base.html

```diff
@@ -26,7 +26,7 @@
 {% block content %}
 <div data-taggings="{{ get_taggings|join:' ' }}"  class="row">
     <div class="col-sm-2 hidden-xs">
-        <nav class="course-menu" id="main-course-menu" aria-labelledby="course-menu-heading">
+        <nav class="course-menu" id="main-course-menu" aria-label="Course Navigation"></nav>
```

Comment:
I think this is wrong. `aria-labelledby="course-menu-heading"` was already good because the course-menu-heading is the title with the word "course" above the course menu (in the desktop screen size). But the page has multiple elements with the same id, so some ids should be changed.

I would use aria-labelledby. If aria-label were used, then note that the current text is not translated and it repeats the word navigation. Since it is the label on a nav element, it should NOT contain the word "navigation". Screen readers already announce the type of elements, thus the label "course navigation" is probably announced as "course navigation navigation". The word "course" is enough to describe the purpose of the navigation landmark and the nav element signals clearly that this is a navigation landmark.

Reply:
Yes, you are right. Using `aria-labelledby` is more convenient in this case